### ISSUE TITLE
Add more masks

### ISSUE THREAD

**Issue by 5e-Cleric** [naturalcrit/homebrewery#3738]
Quite literally

![image](https://github.com/user-attachments/assets/563c126d-f8fa-4d1e-8090-27b49b7ab568)

These i already have made, and i accept requests for more options

I think they provide better options, and make easier a job only possible right now by combining multiple masks, sometimes a pain to manage.

**Comment by calculuschild** [comment]
Is black the image or the text?

**Comment by Gazook89** [comment]
Black has got to be image judging from .0005.png

**Comment by calculuschild** [comment]
Something like 0001 and 0004 at least can already be achieved by just rotating the existing edge masks we have.

The

**Comment by 5e-Cleric** [comment]
As a user, i would prefer to insert and forget, rather than having to mess with the existing code, specially because myself i am not sure if you can even invert a mask

**Comment by calculuschild** [comment]
Insert and forget, exactly. And no need to mess with extra code. We just make a new snippet `ImageMaskFrame` to go along with `imageMaskCenter` and `imageMaskEdge`. Those were all designed very intentionally to be flexible and customizable, so any new masks should follow that design.

It would be identical to `imageMaskCenter` except for this line: 

https://github.com/naturalcrit/homebrewery/blob/9419186e78a0f30c0361968ada24cfe940e0f2c7/themes/V3/Blank/style.less#L302

Becomes

```
mask-image     : var(--wc), linear-gradient(#000 0 0), var(--revealer);
mask-composite : exclude;
```

This results in an inverted mask that behaves like 0005, 0006, and 0007. With the bonus that the user gets access to *all* the existing masks usable as a "frame" without even having to know any inverting code. And, they can rotate / scale / position the mask exactly where they want it, just like all of our other masks, instead of being locked-in to "full column" or "half page" options.

We could even pre-position and scale the masks like we do with Top/Right/Bottom/Left `edge` masks to have a Top/Right/BottomLeft/Full `frame` snippet. Even less thinking on the part of the user. They just get access to more masks.

**Comment by 5e-Cleric** [comment]
> This results in an inverted mask that behaves like 0005, 0006, and 0007. With the bonus that the user gets access to _all_ the existing masks usable as a "frame" without even having to know any inverting code. And, they can rotate / scale / position the mask exactly where they want it. So we can add your new masks. But let's make them usable both as cutouts _and_ frames.
> 

That would work for unique masks, like center masks, would not work for corner or edge masks, since edge only have 1 side and corners are mirrored, so either way i could make some more "square-y" masks.

idk, i don't like symmetrical masks one bit.

**Comment by dbolack-ab** [comment]
Have these been incorporated?

**Comment by 5e-Cleric** [comment]
no, they haven't, there is, to my surprise, a disagreement with calc, so i will be closing this and keeping the masks to myself, until someone asks for more.
